docs/features.md

```diff
@@ -48,4 +48,14 @@ V3 Metadata uses the `ECS_CONTAINER_METADATA_URI` environment variable. Unlike V
+However, in a few cases, this will not work. This is because the Local Endpoints container needs to be able to determine which container a request for V3 metadata came from. Local Endpoints attempts to use the IP address in the request to determine this. If you use the [example Docker Compose file](../examples/docker-compose.yml) with a bridge network, then this IP lookup will work. However, if you use different network settings, then the Local Endpoints will not be able to determine which container a request came from. In this case, set `ECS_CONTAINER_METADATA_URI` to `http://169.254.170.2/v3/containers/{container name}`. The value for `container name` can be any unique substring of your container's name. By setting a custom request URL, the Local Endpoints container can determine which container a request came from.
+
+#### Task Metadata V4
+
+V4 Metadata uses `ECS_CONTAINER_METADATA_URI_V4` environment variable. In most cases, you can set `ECS_CONTAINER_METADATA_URI_V4` to `http://169.254.170.2/v3`. Similarly, in a few cases when Local Endpoints container doesn't know which container a request came from, you have to set `ECS_CONTAINER_METADATA_URI_V4` to `http://169.254.170.2/v3/containers/{container name}`. Please see the description above on V3 to understand why you'll need to specify the container name in the path.
+
+However, compared to V3, V4 includes additional network metadata when querying the task metadata endpoint (see [here](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/task-metadata-endpoint-v4.html)). Please refer to this [example](../examples/v4) if you want to include those additional V4 metadata. You can use the generic metadata injection feature (described below) to add the additional metadata fields included in V4.
+
+#### Generic Metadata Injection
+
+As mentioned above in the previous section, to inject generic metadata, you'll need to have those additional metadata in JSON files. Then specify paths for the JSON files by using `CONTAINER_METADATA_PATH` and `TASK_METADATA_PATH` environment variables. More specifically, `CONTAINER_METADATA_PATH` is the metadata for each container, which will override their counterparts in the normal response. Also, `TASK_METADATA_PATH` is for task level metadata, which is used only for overriding the top level fields in the task metadata response. If you specify both `CONTAINER_METADATA_PATH` and `TASK_METADATA_PATH`, then the override container metadata will be injected into the override task metadata response. See example for overriding task metadata response [here](../example/generic).
```

Comment:
> If you specify both `CONTAINER_METADATA_PATH` and `TASK_METADATA_PATH`, then the override container metadata will be injected into the override task metadata response.

This doesn't quite convey what I want. Two sentences/ideas are needed here.
- The metadata from `CONTAINER_METADATA_PATH` will be included in the `Containers` section of the Task Metadata response.
- Metadata that you inject from these files overrides the normal values that would have otherwise been returned in the response.